### PR TITLE
Fixing the Bounding Rectangle Selection on reload

### PR DESCRIPTION
**Pull request by jinman**
Thanks to @JAlvezzz, I am merging his changes after testing and creating PR.

### CODE REVIEW

**Review by agentcooper — CHANGES_REQUESTED** [review]
Could you also please use Prettier to format? CI shows incorrect formatting as an error.

File: packages/react-pdf-highlighter/src/components/MouseSelection.js [hunk 1]
```diff
@@ -81,15 +81,22 @@ class MouseSelection extends Component<Props, State> {
...
+        if (window.scrollY > 0) {
+          realContainerBoundingRect.top += window.scrollY;
+        }
       }
 
       return {
-        x: pageX - containerBoundingRect.left + container.scrollLeft,
-        y: pageY - containerBoundingRect.top + container.scrollTop
+        x: pageX - realContainerBoundingRect.left + container.scrollLeft,
+        y: pageY - realContainerBoundingRect.top + container.scrollTop
```
[the remark this answers is not in the document]
Comment (jinman) [reply packages/react-pdf-highlighter/src/components/MouseSelection.js]: I tried and it works that but was not sure whether I missing some test cases. So I reverted back to @JALvezzz's checkin assuming that he has done more comprehensive testing. Anyway, I checkedin with the fix as per your recommendation.

**Comment by jinman** [comment]
@agentcooper can we approve this change for the main build. Does this work? Thank you!

**Comment by jinman** [comment]
@agentcooper Any idea this can be merged?

**Comment by agentcooper** [comment]
Thanks!

**Comment by jinman** [comment]
@agentcooper do you know when will this released and merged in npm v3.3.1 ?

**Comment by agentcooper** [comment]
Published in 3.4.0.

**Pull request merged**